plugins/scaffolder/api-report.md

```diff
@@ -264,6 +264,7 @@ export type NextRouterProps = {
   };
   groups?: TemplateGroupFilter[];
   transformErrors?: ErrorTransformer;
+  noHtml5Validate?: boolean;
```

Comment:
Is this something we would like for all templates? If not, have you considered adding this as a [template](https://backstage.io/docs/features/software-templates/adding-templates) optional configuration?